src/software/geom/polygon.cpp

```diff
@@ -76,14 +76,15 @@ Polygon Polygon::expand(double expansion_amount) const
     return Polygon(expanded_points);
 }
 
-Polygon Polygon::fromSegment(const Segment& segment, const double radius)
+Polygon Polygon::fromSegment(const Segment& segment, const double radius_parallel,
+                             const double radius_normal)
 {
     /*   The Polygon is constructed as follows:
      *
      *        start_l                start_r
      *           +----------+----------+
      *           |          |          |
-     *           |          | radius   |
+     *           |          | radius_n |
```

Comment:
it looks like radius_p and radius_n are exchanged here (that is radius_p should be in the same line as start and end)